apps/code-examples/src/app/code-examples/popovers/popover/basic/popover-demo.component.html

```diff
@@ -1,13 +1,13 @@
 <button
-  class="sky-btn sky-margin-inline-compact"
+  class="sky-btn sky-btn-default sky-margin-inline-compact"
```

Comment:
Does `sky-margin-inline-compact` need to change to the "modern" CSS class? Or will you handle that in a subsequent PR?

Reply:
That's been changed in this PR: https://github.com/blackbaud/skyux/pull/161/files#diff-1f41a0be4ff900f211dc873c2ab15d7994809bfec19a2f1d4fd4332ecbd63c6f